### PR TITLE
Fixed EZP-28718: Embedded images should be handled as images instead of plain embed object after migrating from XmlText to RichText

### PR DESCRIPTION
**Pull request by vidarl**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | [EZP-28718](https://jira.ez.no/browse/EZP-28718)
| Versions      | 1.7

Doc changes related to https://github.com/ezsystems/ezplatform-xmltext-fieldtype/pull/39

PR should not be merged before new version of ezplatform-xmltext-fieldtype is released

### CODE REVIEW

**Comment by vidarl** [comment]
@DominikaK : please have a look

File: docs/migrating/migrating_from_ez_publish_platform.md [hunk 1]
```diff
@@ -162,6 +162,38 @@ For migrating content from the XmlText format to the new RichText format, a migr
...
+
+    [RelationGroupSettings]
+    ImagesClassList[]
+    ImagesClassList[]=image
+
+!!! note
+
+    Version prior to v1.5.1 of the migration script would fail to convert embedded images correctly. If you have a database which you already have converted with a old version, you may rerun convert-to-richtext command with the following options:
+
+    `php app/console ezxmltext:convert-to-richtext --fix-embedded-images-only --image-content-types=image,thumbnail,profile -v`
```
Comment (vidarl) [inline docs/migrating/migrating_from_ez_publish_platform.md]: I guess this block maybe won't appear in the "note" box ? is that possible?

**Comment by vidarl** [comment]
Rebased, against 1.13 instead of 1.7

File: docs/migrating/migrating_from_ez_publish_platform.md [hunk 2]
```diff
@@ -196,6 +196,40 @@ Steps here should only be done once you are ready to move away from legacy and L
...
+
+```
+[RelationGroupSettings]
+ImagesClassList[]
+ImagesClassList[]=image
+```
+
+!!! note
+
+    Version of the migration script prior to v1.5.1 would fail to convert embedded images correctly. If you have a database which you have already converted with an old version, you may rerun the `convert-to-richtext` command with the following options:
```
Comment (andrerom) [inline docs/migrating/migrating_from_ez_publish_platform.md]: `v1.5.1` => `ezplatform-xmltext-fieldtype v1.6.0`  

1. to be more specific
2. as we are talking about prior (1.5.1 is also affected)

Comment (vidarl) [reply docs/migrating/migrating_from_ez_publish_platform.md]: Fixed

**Comment by DominikaK** [comment]
@vidarl can we merge it now that https://github.com/ezsystems/ezplatform-xmltext-fieldtype/releases/tag/v1.6.0 is out?
Does this apply to 1.13 only, or to all more recent versions as well?

**Pull request merged**